cluster/v1alpha1/types.go

```diff
@@ -301,32 +301,79 @@ const (
 
 // PrioritizerConfig represents the configuration of prioritizer
 type PrioritizerConfig struct {
+	// Name will be deprecated in v1beta1 and replaced by PrioritizerScoreCoordinate.Builtin.
```

Comment:
note that this is going to require a conversionwebhook.

Reply:
deprecated or removed?

Reply:
modify to removed.